test/pummel/test-keep-alive.js

```diff
@@ -75,8 +75,8 @@ function runAb(opts, callback) {
   });
 }
 
-server.listen(common.PORT, function() {
-  runAb({ keepalive: true }, function(reqSec) {
+server.listen(common.PORT, () => {
+  runAb({ keepalive: true }, (reqSec) => {
     keepAliveReqSec = reqSec;
     console.log('keep-alive:', keepAliveReqSec, 'req/sec');
```

Comment:
I think we can remove all the `console.log`s and the commented `console.log` in this file.